### PR TITLE
add: stardust-comet

### PR DESCRIPTION
**Pull request by Owen-sz**

### CODE REVIEW

File: anda/stardust/comet/stardust-comet.spec [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+%description
+Annotate things in Stardust
+
+%prep
+%autosetup -n comet-%commit
+%cargo_prep_online
+
+%build
+%cargo_build
```
Comment (madonuko) [inline anda/stardust/comet/stardust-comet.spec]: don't need this

File: anda/stardust/comet/stardust-comet.spec [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Name:           stardust-comet
+Version:        %commit_date.%shortcommit
+Release:        1%?dist
+Summary:        Annotate things in Stardust.
+URL:            https://github.com/StardustXR/comet
+Source0:        https://github.com/StardustXR/comet/archive/%commit/comet-%commit.tar.gz
+License:        MIT
+BuildRequires:  cargo cmake anda-srpm-macros cargo-rpm-macros mold
+Requires:       libgcc glibc
```
Comment (madonuko) [inline anda/stardust/comet/stardust-comet.spec]: don't need this

File: anda/stardust/comet/stardust-comet.spec [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+%global shortcommit %(c=%{commit}; echo ${c:0:7})
+# Exclude input files from mangling
+%global __brp_mangle_shebangs_exclude_from ^/usr/src/.*$
+
+Name:           stardust-comet
+Version:        %commit_date.%shortcommit
+Release:        1%?dist
+Summary:        Annotate things in Stardust.
+URL:            https://github.com/StardustXR/comet
+Source0:        https://github.com/StardustXR/comet/archive/%commit/comet-%commit.tar.gz
```
Comment (madonuko) [inline anda/stardust/comet/stardust-comet.spec]: dry

File: anda/stardust/comet/update.rhai [hunk 4]
```diff
@@ -0,0 +1,7 @@
+if filters.contains("nightly") {
```
Comment (madonuko) [inline anda/stardust/comet/update.rhai]: don't need this

File: anda/stardust/comet/stardust-comet.spec [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+License:        MIT
+BuildRequires:  cargo cmake anda-srpm-macros cargo-rpm-macros mold
+Requires:       libgcc glibc
+
+Provides:       comet
```
Comment (madonuko) [inline anda/stardust/comet/stardust-comet.spec:16]: add `Packager:`

File: anda/stardust/comet/stardust-comet.spec [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+%description
+Annotate things in Stardust
```
Comment (madonuko) [inline anda/stardust/comet/stardust-comet.spec]: full stop here, remove full stop from Summary

**Review by madonuko — COMMENTED** [review]
also you accidentally assigned me and lea instead of asking for a review

File: anda/stardust/comet/stardust-comet.spec (hunk 5 above)
Comment (Owen-sz) [reply anda/stardust/comet/stardust-comet.spec:16]: Done, is that set correctly?

File: anda/stardust/comet/stardust-comet.spec (hunk 2 above)
Comment (madonuko) [reply anda/stardust/comet/stardust-comet.spec]: ?

File: anda/stardust/comet/stardust-comet.spec (hunk 5 above)
Comment (madonuko) [reply anda/stardust/comet/stardust-comet.spec:16]: lgtm

File: anda/stardust/comet/update.rhai (hunk 4 above)
Comment (madonuko) [reply anda/stardust/comet/update.rhai]: remove the if statement

**Review by madonuko — CHANGES_REQUESTED** [review]

**Review by madonuko — DISMISSED** [review]

**Review by madonuko — DISMISSED** [review]

**Review by madonuko — APPROVED** [review]

**Comment by raboneko** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|f39|[](https://github.com/terrapkg/packages/pull/2099)|
|✅|f40|[](https://github.com/terrapkg/packages/pull/2100)|
|✅|f41|[](https://github.com/terrapkg/packages/pull/2101)|
|✅|el9|[](https://github.com/terrapkg/packages/pull/2102)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport) and see the [Github Action logs](https://github.com/terrapkg/packages/actions/runs/10943076375) for details

**Pull request merged**